saleor/graphql/shop/types.py

```diff
@@ -117,8 +118,10 @@ def resolve_authorization_keys(self, info):
         return site_models.AuthorizationKey.objects.all()
 
     def resolve_countries(self, info):
+        taxes = {vat.country_code: vat for vat in VAT.objects.all()}
```

Comment:
This looks like something that could be extracted outside of the resolver, to be calculated once, not every time query involving countries is being executed.

Reply:
We could leave it as it is - it's a single database query to fetch a constant number of items (28). Also building this relatively small dictionary is rather efficient in Python and shouldn't impact the performance. I'd instead debug the entire `shop` query in our client apps and see how fast it is or how many queries in total it performs.